js/bootstrap-slider.js

```diff
@@ -204,8 +204,26 @@
 			linear: {
 				toValue: function(percentage) {
 					var rawValue = percentage/100 * (this.options.max - this.options.min);
-					var value = this.options.min + Math.round(rawValue / this.options.step) * this.options.step;
+					if (this.options.ticks_positions) {
```

Comment:
This large block of code looks very similar to lines 240-256...Anyway we can refactor that logic into a single method?